### DISCUSSION TITLE
Suggest some Idea/ Content which we can add on Landing page of our website !

### DISCUSSION THREAD

**Discussion by rohan-kulkarni-25** [Coders-Evoke-Community/CodersEvoke_website#13] [Ideas]
Do suggest ideas which we can implement in our website for the landing page !

**Comment by nirban256** [comment]
You can use glassmorphism style for the main content and make a fixed navbar when scrolling is done. Hope , it helps🙂

**Reply by rohan-kulkarni-25** [reply]
Thanks, @nirban256 !!

**Comment by sarthakjdev** [comment]
Use fonts like "Alata" , "Josefin sans " , "Manrope" . 

Moreover use gradients if possible .

**Comment by Ritwik880** [comment]
Hey @rohan-kulkarni-25 what if we implement some animation like fade up and fade in, whenever the user scrolls down then only the content will get render.

**Comment by SHIVSB** [comment]
You need to reduce the padding around the home , about, and other such buttons. Another thing you can do is making a dedicated section in case someone has got any ideas about making a project and needs a team to bring it to reality.
